package.json

```diff
@@ -11,12 +11,14 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "seed": "node server/db/seed.js"
+    "seed": "node server/db/seed.js",
+    "start": "nodemon server/index.js"
```

Comment:
This looks good for your script!

Reply:
@rlnolan15 Thanks. I will merge the PR now.